ftw/solr/tests/test_handlers.py

```diff
@@ -294,6 +294,23 @@ def test_add_without_attributes_calls_add_and_extract(self):
             'application/vnd.openxmlformats-officedocument.wordprocessingml.document'
         )
 
+    def test_add_with_empty_list_attributes_calls_add_and_extract(self):
+        self.handler.add([])
```

Comment:
To the reviewer: this test is a copy of https://github.com/4teamwork/ftw.solr/pull/155/files#diff-631f017b550cd5f52518f72f344230c4R280 but with an empty list instead of `None`.